addons/crm/models/crm_lead.py

```diff
@@ -2108,6 +2108,43 @@ def _track_subtype(self, init_values):
             return self.env.ref('crm.mt_lead_lost')
         return super()._track_subtype(init_values)
 
+    def _initial_values_creation_message(self):
+        self.ensure_one()
+        message_bits = []
+        name_fields = ["partner_name", "contact_name"]
+        address_fields = ["street", "street2", "zip", "city", "state_id", "country_id"]
+
+        lead_values = {}
+        for info_field in name_fields + address_fields + ["function", "website"]:
+            if val := self[info_field]:
+                lead_values[info_field] = val.display_name if info_field in {"state_id", "country_id"} else val
+
+        if any(lead_values.get(info_field) for info_field in name_fields):
+            message_bits.append(
+                Markup("<i class='fa fa-user fa-fw me-1'/><b>%s</b>" % ", ".join(
+                    lead_values.get(info_field) for info_field in name_fields if lead_values.get(info_field)
+                ))
+            )
+        if lead_function := lead_values.get("function"):
```

Comment:
If it's only some icon change, maybe we could have a mapping field_name -> icon to ease code ?